organisms/organisation-unit-tree/new-features/select-by-group-level.feature

```diff
@@ -0,0 +1,56 @@
+# Generic
+
+Feature: Selection of descendant levels and/or groups inside an org unit
+
+  Background:
+    Given an org unit tree with root org unit 'Sierra Leone'
+    And descendants inside 'Sierra Leone' with multiple groups and levels
+
+  Scenario: Selection of a level or group inside an org unit
```

Comment:
I assume that, similar to the "Select all descendants" mode, all selected descendants (selected either by group or level) should have an indeterminate state and should not be deselect-able?

Reply:
This is a tricky one, I'm leaning towards only using a visual indication of the group/level selection _that will be made at runtime,_ not actually changing the appearance of the org.unit nodes. As discussed, the level/group selection will actually be done at runtime on the BE (when performing an analytical request, for example). If we want to put indeterminate states on all of the nodes in the tree we will need to run the group/level selection on the FE too. What do you think?

Reply:
It's not really tricky to do that on the FE. If group/level selection is active, the FE already knows which group or level and can therefore just highlight all the descendants. But I think it's ok to not show this in the org unit tree itself